src/main/java/ca/uwaterloo/drinkmasterapi/sql/creation/admin.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE admin (
+                      id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                      username VARCHAR(255) NOT NULL UNIQUE,
+                      email VARCHAR(255) NOT NULL UNIQUE,
+                      password VARCHAR(255) NOT NULL,
+                      machine_id BIGINT,
```

Comment:
the latest design is: after admin login, system will provide a drop down list of machine id for them, one (admin)to many(machine) , this need to be changed, need discuss it first